third_party/Elecard/MpegAudioEncLayer2/mpegaudio.cpp

```diff
@@ -528,7 +528,7 @@ static void compute_bit_allocation(MpegAudioContext *s,
     for(i=0;i<s->sblimit;i++) {
         incr = alloc[0];
         current_frame_size += incr * s->nb_channels;
-        alloc += 1 << incr;
+        alloc += 1i64 << incr;
```

Comment:
Use an 'LL' suffix rather than the MS non-standard i64 suffix.

Reply:
Done